eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs

```diff
@@ -63,9 +68,9 @@ import Lens.Micro
 -- being total. Do not change it!
 --------------------------------------------------------------------------------
 
-type instance TranslationContext (ConwayEra c) = ConwayGenesis c
+type instance TranslationContext (ConwayEra c) = ConwayGenesis (ConwayEra c)
 
-instance Crypto c => TranslateEra (ConwayEra c) NewEpochState where
+instance forall c. (Crypto c, EraPParams c) => TranslateEra (ConwayEra c) NewEpochState where
```

Comment:
This constraint is totally wrong. It essentially says that `c` is both `Crypto` and an `Era` at the same time. If you ever see yourself mixing up `c` and `era` at the same time it should be a red flag that something is wrong

Also `forall` is totally redundant
```suggestion
instance Crypto c => TranslateEra (ConwayEra c) NewEpochState where
```